swagger.yaml

```diff
@@ -412,8 +509,65 @@ definitions:
         description: |
           The name of the filter, as it is definined in Skipper
       args:
+        description: The arguments for the filter, can be string or float64
+        type: array
+        items:
+          type: string
+  Host:
+    required:
+      - id
+      - name
+    properties:
+      id:
+        type: string
+        format: int32
+      name:
+        type: string
+    example: |
+      {
+        "id": "1",
+        "name": "zalando.de"
+      }
+  PathIn:
+    required:
+      - uri
```

Comment:
Yes, empty URI should be prohibited. How can I do it in swagger spec? Just a description?